### DISCUSSION TITLE
New documentation on the wiki: How to add the first user (Admin) to Redash without using the web interface

### DISCUSSION THREAD

**Discussion by justinclift** [getredash/redash#6915] [Announcements]
Just added a new document to the wiki for people deploying Redash:

&nbsp; &nbsp; https://github.com/getredash/redash/wiki/How-to-add-the-first-user-(Admin)-to-Redash-without-using-the-web-interface

This might be of use to anyone wanting to deploy Redash in an automated fashion, where they'd need to add the initial Admin user prior to starting the web interface.

**Comment by eradman** [comment]
It's a lot easier than this: the manage command can create users, including the Admin

```
docker compose exec server ./manage.py users create_root eradman@eradman.com Redash --password ******
```

**Reply by justinclift** [reply]
Awesome!  I'll update the wiki page with that approach, probably tomorrow. :smile:
